lib/nifty_services/base_service.rb

```diff
@@ -188,7 +188,9 @@ def error!(status, message_key, options = {})
     end
 
     def valid_object?(record, expected_class)
-      record.present? && record.is_a?(expected_class)
+      blank = record.respond_to?(:empty?) ? !!record.empty? : !record
```

Comment:
Maybe `record.is_a?(expected_class)` will be enough here, if its nil, false, or whatever, will not be `expected_class`, or it can be depending of user want, better no?